webdriver/tests/bidi/script/call_function/remote_values.py

```diff
@@ -39,3 +39,129 @@ async def test_remote_value_promise(bidi_session, top_context, await_promise):
         assert result == {"type": "number", "value": 42}
     else:
         assert result == {"type": "promise"}
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("await_promise", [True, False])
+async def test_window_context_top_level(bidi_session, top_context,
+                                        await_promise):
+    function_declaration = f"() => window"
+    if await_promise:
+        function_declaration = "async" + function_declaration
+
+    result = await bidi_session.script.call_function(
+        function_declaration=function_declaration,
+        await_promise=await_promise,
+        target=ContextTarget(top_context["context"]),
+        serialization_options=SerializationOptions(max_object_depth=1),
+    )
+
+    recursive_compare(
+        {
+            "type": "window",
+            "value": {
+                "context": top_context["context"]
+            }
+        }, result)
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("await_promise", [True, False])
+async def test_window_context_same_id_after_cross_origin_navigation(
```

Comment:
Similar to other navigation tests we should use parametrize here to trigger a same origin and cross origin navigation. In both cases the id need to stay the same. Sorry that I haven't explicitly mentioned that before.